modules/oe_content_reference/README.md

```diff
@@ -0,0 +1,9 @@
+# OpenEuropa Reference Field
+
+This module provides the Reference Field storage field, by enabling it you will be able to add reference
+fields to your content types. Reference fields will be automatically styled when using OpenEuropa Theme.
```

Comment:
Let's use the description below, so we don't confuse users, as they might thing it has something to do with referencing entities.

> This module provides the storage configuration for a reference code field. A reference code field is a simple text field that can be used to store codes, IDs and other reference-like information. This field is used on several corporate content types, so it is packaged in its own, independent, module.

Reply:
done